### ISSUE TITLE
The program opens and closes when you copy it to the folder of a game

### ISSUE THREAD

**Issue by Japacho** [x360ce/x360ce#292]
When trying to copy the program, it opens and closes frequently and does nothing else, it is as though it were in an infinite bug, This happens with the game guitar hero

**Comment by Squall-Leonhart** [comment]
application must be run elevated in a program files location.

**Comment by passingover** [comment]
I am having the same issue with portal 2

I am running it from the game directory where the exe is

program files/valve/portal2
(portal2.exe is there and x360ce_64.exe is there as well)

I tried updating net framework based off other suggestions to no avail.

help me please

**Comment by Squall-Leonhart** [comment]
Source engines require the dll be created in the location of the inputmanager.dll and that the settings file be in programdata location.

**Comment by XstreamGit** [comment]
I have the same problem with Portal 2, no matter where I place the dll file

**Comment by stevefcb** [comment]
open as admin solves the problem

**Comment by XstreamGit** [comment]
Nope, still crashes when launching the game (before getting into the main menu)

**Comment by EvoJocys** [comment]
This report is about the 3.x line, where x360ce was an xinput DLL copied into each game's folder. The current 4.x line works differently: it creates a virtual Xbox 360 controller that every game sees, with nothing copied into game folders. Game-specific problems from the DLL era do not carry over.

The latest release is 4.21 (September 2026): https://github.com/x360ce/x360ce/releases

Closing as part of a tracker clean-up after the 4.21 release. If it still happens on 4.21, reopen this issue with the version and what you see.
